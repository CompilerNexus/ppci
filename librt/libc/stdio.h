#ifndef STDIO_H
#define STDIO_H

void printf(char*);
void bsp_putc(char);

#endif

