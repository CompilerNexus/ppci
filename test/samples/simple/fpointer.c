
#include <stdio.h>

void do1(int x)
{
  printf("do1 %d\n", x);
}

void do2(int x)
{
  printf("do2 %d\n", x);
}

void main_main()
{
  void (*f)(int x);
  f = do1;
  f(30);
  f = &do2;
  f(33);
}
