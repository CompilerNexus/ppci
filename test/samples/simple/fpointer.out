do1 30
do2 33
